df.raws.xml

```diff
@@ -25,6 +25,20 @@
         </virtual-methods>
     </class-type>
 
+    <class-type type-name='soundst'>
+        <stl-string name='token'/>
+        <int32_t name='index'/>
+        <stl-vector name='current_definition'>
+            <stl-string/>
```

Comment:
This needs to be `<pointer type-name='stl-string'/>`, since Toady's `stringvectst` stores pointers rather than whole objects (for Reasons™).

Come to think of it, we should probably just create a formal `stringvectst` class (see `g_src/enabler.h`) so that if Toady eventually _does_ change it, we'll only have to fix it in one place.